### PR TITLE
Update mobile-faq.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor grammar, punctuation, and style updates.

### CODE REVIEW

File: source/mobile/mobile-faq.rst [hunk 1]
```diff
@@ -4,26 +4,26 @@ Mobile Apps FAQ
...
+At the moment, we only support connecting to one server at a time; however, we are aware that this is one of the `top feature requests <https://mattermost.uservoice.com/forums/306457-general/suggestions/10975938-ios-and-android-apps-should-allow-multiple-server>`__ for the mobile app. We are currently investigating some technical challenges, such as how to handle push notifications coming from multiple servers. To follow our progress on this feature, you can join the `RN: Multi-Server <https://community.mattermost.com/core/channels/rn-multi-server-suppot>`_ channel on our community server.
 
-As a workaround, you can install both the released "Mattermost" app and sign up to be a `tester <https://github.com/mattermost/mattermost-mobile/blob/master/README.md#testing>`__ for the "Mattermost Beta" app. This allows you to connect and log in to a different server from each app.
+As a workaround, you can install the released Mattermost mobile app and sign up to be a `tester <https://github.com/mattermost/mattermost-mobile/blob/master/README.md#testing>`__ for the Mattermost Beta mobile app. This allows you to connect and log in to a different server from each app.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:9]: @esethna, is there still a Mattermost Beta mobile app available as per this section? If so, are there other steps involved after signing up?

File: source/mobile/mobile-faq.rst [hunk 2]
```diff
@@ -4,26 +4,26 @@ Mobile Apps FAQ
...
 
 Is there a tablet version of the mobile apps?
 ---------------------------------------------
 
-“Mattermost Classic” mobile apps support tablets.
+Mattermost Classic mobile apps support tablets.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:14]: @esethna, is this still the correct name?

File: source/mobile/mobile-faq.rst [hunk 3]
```diff
@@ -4,26 +4,26 @@ Mobile Apps FAQ
...
 
-Our second generation mobile apps (“Mattermost”) have beta support for tablets.
+Our second generation mobile apps (Mattermost) have beta support for tablets.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:16]: @esethna, is this still accurate?

File: source/mobile/mobile-faq.rst [hunk 4]
```diff
@@ -4,26 +4,26 @@ Mobile Apps FAQ
...
+The permanent sidebar is on by default for tablet-sized devices, but can be disabled from **Settings > Display > Sidebar > Permanent Sidebar**. When disabled, the sidebar behaves similarly to mobile devices where the user must open it using the button in the top-left corner of the screen.
 
 How is data handled on mobile devices after a user account is deactivated?
 --------------------------------------------------------------------------
 
-App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute the system logs the user out, and as a result all app data is wiped from the device.
+App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute of deactivation the system logs the user out. Thereafter all app data is wiped from the device.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:26]: @DSchalla / @esethna, confirming that this is correct.

File: source/mobile/mobile-faq.rst [hunk 5]
```diff
@@ -74,38 +74,38 @@ The following post metadata is sent in all push notifications:
...
 
 What are my options for securing the mobile apps?
 -------------------------------------------------
 
-The following options for security mobile application deployments are available:
+The following options for secure mobile app deployments are available:
 
 1. Securing network connection to mobile apps
   - Use HTTPS and WSS network connections to encrypt transport.
   - Use of a mobile VPN client on mobile devices to establish secure connection to Mattermost server within private network.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:88]:
```suggestion
  - Use of a mobile VPN client on mobile devices to establish secure connection to Mattermost server within a private network.
```

Comment (justinegeffen) [reply source/mobile/mobile-faq.rst:88]: @esethna / @DSchalla: do we recommend any specific mobile VPN clients?

File: source/mobile/mobile-faq.rst [hunk 6]
```diff
@@ -139,14 +139,14 @@ How do I host the Mattermost push notification service?
 
 First, you can use the :doc:`Mattermost Hosted Push Notification Service (HPNS) <mobile-hpns>`. Organizations can also `host their own push proxy server instead <https://developers.mattermost.com/contribute/mobile/push-notifications/service/>`_. This is applicable when you want to:
 
-1. customize the `Mattermost mobile apps <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_,
-2. deploy your own push notification service, or
-3. repackage the mobile apps with BlueCedar or AppDome, which some organizations have successfully deployed with but is not officially supported.
+1. Customize the `Mattermost mobile apps <https://developers.mattermost.com/contribute/mobile/build-your-own/>`_;
+2. Deploy your own push notification service, or
+3. Repackage the mobile apps with BlueCedar or AppDome (both of which are not officially supported but have been successfully deployed by some organizations).
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:144]: @DSchalla / @esethna: confirming that this is still correct.

File: source/mobile/mobile-faq.rst [hunk 7]
```diff
@@ -157,19 +157,19 @@ All files in the ``/assets/base`` folder can be overriden as needed without conf
 2. Copy any files or folders that you wish to replace from ``/assets/base`` into ``/assets/override``.
 3. Make your changes to the files in ``/assets/override``.
 
-When you compile the app or run ``make dist/assets``, the contents of those two folders will be merged with files in ``/assets/override``, taking precedence in the case of any conflicts. For binary files such as images, an overridden file will completely replace the base version, while json files will be merged so that fields not set in the overridden copy use the base version.
+When you compile the app or run ``make dist/assets``, the contents of those two folders will be merged with files in ``/assets/override``, taking precedence in the case of any conflicts. For binary files such as images, an overridden file will completely replace the base version, while ``JSON`` files will be merged so that fields not set in the overridden copy use the base version.
 
 For a more specific example of how to use this feature, see the following section.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:162]: @esethna, do you know which doc/section this should link to? Or should we remove this line?

File: source/mobile/mobile-faq.rst [hunk 8]
```diff
@@ -229,7 +229,7 @@ How do I connect users across internal and external networks?
 
 By setting up global network traffic management, you can send a user to an internal or external network when connecting with a mobile app. Moreover, you can have two separate layers of restrictions on internal and external traffic, such as:
 
- - In the internal network, deploy on a private network via per device VPN
+ - In the internal network, deploy on a private network via per device VPN.
  - In the external network, deploy with `TLS mutual auth <https://docs.mattermost.com/deployment/ssl-client-certificate.html>`__ with an NGINX proxy, and `client-side certificates <https://docs.mattermost.com/deployment/certificate-based-authentication.html>`__ for desktop and iOS.
  
 Many services such as Microsoft Azure provide options for `managing network traffic <https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-overview>`__, or you can engage a services partner to assist.
```
Comment (justinegeffen) [inline source/mobile/mobile-faq.rst:235]: Is the service partner external, internal to the organization, or from Mattermost?

File: source/mobile/mobile-faq.rst (hunk 1 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:9]: @justinegeffen yes, there is a Mattermost Beta mobile app.

File: source/mobile/mobile-faq.rst (hunk 2 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:14]: yes, the name of the previous apps is `Mattermost Classic`

File: source/mobile/mobile-faq.rst (hunk 4 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:26]: Yes, the only caveat is that if the user is logged in but not using the app, they'll get logged out the next time the open the app and therefore all data is wiped.

File: source/mobile/mobile-faq.rst (hunk 5 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:88]: Never recommend a third party option unless there is some sort of partnership (maybe) which we don't have for VPN clients.

File: source/mobile/mobile-faq.rst (hunk 6 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:144]: correct

File: source/mobile/mobile-faq.rst (hunk 7 above)
Comment (enahum) [reply source/mobile/mobile-faq.rst:162]: all this info is in the developers doc if I'm not mistaken

File: source/mobile/mobile-faq.rst (hunk 3 above)
Comment (esethna) [reply source/mobile/mobile-faq.rst:16]: Yes it is

File: source/mobile/mobile-faq.rst (hunk 7 above)
Comment (justinegeffen) [reply source/mobile/mobile-faq.rst:162]: Thanks, @enahum! I've taken a look through here: https://developers.mattermost.com/contribute/mobile/ but couldn't pin down the exact section (maybe wasn't looking hard enough). Is a general link sufficient?

**Pull request merged**